### DISCUSSION TITLE
How to increase "Results per page"

### DISCUSSION THREAD

**Discussion by DKB0512** [rayanfer32/TgindexPro#20] [Q&A]
I just wanna ask that if it is possible to increase per page result? like its 20 currently but can I increase it upto 30 or 50?
I also want to ask that If I share any mkv in my group that is connected with tgindex, can I do something to get a thumbnail for it? Like currently its assigning a random color to it.

**Comment by pseudokawaii** [comment]
It is possible in the [main repo](https://github.com/odysseusmax/tg-index). By passing the `RESULTS_PER_PAGE` env variable.

**Reply by DKB0512** [reply]
Not working! I applied RESULTS_PER_PAGE as env variable but it still shows 20

**Reply by pseudokawaii** [reply]
@DKB0512 on the main repo?

**Reply by DKB0512** [reply]
Since I cloned this on repl using your method, I tried that on this repo

**Reply by pseudokawaii** [reply]
@DKB0512 that feature isn't available on this repo yet. You have to redeploy the main repo. But the main repo got some new commits that broke the Replit support, so use [this repo](https://github.com/culturecloud/tg-index/tree/replit-patch) instead to deploy on Replit, it has some new fixes that the original repo doesn't.
